pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -191,7 +191,13 @@ public void processSqlWithMultiStageQueryEngineGet(
           executeSqlQuery(requestJson, makeHttpIdentity(requestContext), true, httpHeaders, true);
       asyncResponse.resume(getPinotQueryResponse(brokerResponse));
     } catch (WebApplicationException wae) {
-      asyncResponse.resume(wae);
+      Response response = wae.getResponse();
```

Comment:
Trying to understand the difference here. Should we always `resume` on the response instead of the exception?

Reply:
Just realised that we do not have Exception Mapper for Broker REST API's . Making that change